tools.yml

```diff
@@ -2309,7 +2291,8 @@ tools:
     cores: 4
     mem: 7.6
   toolshed.g2.bx.psu.edu/repos/iuc/scanpy_remove_confounders/scanpy_remove_confounders/.*:
```

Comment:
Your config doesn't override `cores` or `mem` unless I am missing it, so it would still have gotten the `9` cores (and then `mem = 9 * 3.8` from your default rule) from the shared DB? I think your env vars restrict it to 4 cores but it's allocated 9?